brave-intl/publishers
# List for channels which need approval 🏷

## List for channels which need approval
![Screen Shot 2019-04-19 at 3 35 13 PM](https://user-images.githubusercontent.com/4713771/56440862-dfd08500-62b8-11e9-8a06-fd90f12fcc3d.png)


#### Features

* This is meant as a quick fix to make it easier to see channels that need approval. 
* We may be removing the need to approve websites altogether, see [discussion](https://bravesoftware.slack.com/archives/C0KHGGEUW/p1555000957102300?thread_ts=1554957966.091700&cid=C0KHGGEUW)

#### How To Test

1. As a publisher add a channel
2. In rails console `channel.update(verification_status: 'awaiting_admin_approval')`
3. As an admin, navigate to the channel approvals list, see the pending approval

#### Pull Request Checklist
